src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml

```diff
@@ -28,6 +28,19 @@ imports:
 parameters:
     sylius_core.public_dir: "%kernel.project_dir%/web"
 
+framework:
```

Comment:
This config should be moved to `src/Sylius/Bundle/CoreBundle/Resources/config/app/messenger.yaml`